intro-Cloud/title.tex

```diff
@@ -5,7 +5,7 @@
 \vspace*{6\baselineskip}
 
 \Huge \strong{Cloud Tutorial}\\
-\LARGE Last updated: DATEPLACEHOLDER
+\LARGE Last updated: May  21 2021
```

Comment:
You should keep `DATEPLACEHOLDER` here, the rendered PDF should automatically get the correct date when the PR is merged?

Reply:
@boegel ah ok, but in our current pdf it was not replaced https://hpcugent.github.io/vsc_user_docs/pdf/intro-Cloud.pdf , I will check again

Reply:
That should be fixed now by the change you made to `add-timestamp.sh`, I think...